### DISCUSSION TITLE
I'm not able to push to github using HTTPS password authentication

### DISCUSSION THREAD

**Discussion by Kah-S** [git-for-windows/git#3623] [Q&A]
it shows up:
![a](https://user-images.githubusercontent.com/97122615/148829541-264b6d72-5399-478e-954f-3a7d6c180a1f.png)
I checked the name and password many times but it wasn´t working
I tried to create token to login but it did not work either

**Comment by dscho** [comment]
The error message is quite clear, no?

> Support for password authentication was removed

Did you follow the link to the blog post for more information?
